mono/mini/interp/transform.c

```diff
@@ -984,6 +984,9 @@ emit_store_value_as_local (TransformData *td, MonoType *src)
 static gboolean
 interp_is_bb_start (TransformData *td, InterpInst *start, InterpInst *end)
 {
+	// Only single basic blocks are inlined.
+	if (td->inlined_method)
+		return FALSE;
```

Comment:
I don't think this is needed, il_offsets should be correct. I might have to update this method at some point since its looks fishy

Reply:
Ah, this is also after IL transformed to IR, and inlining finished?
I missed the distinction in the code where it moves from IL to IR.

Reply:
I undid it.

Reply:
We could do inlining but the assigned il_offset to an instruction is always from the parent method, so it's fine to access is_bb_start with those offsets (it might be redundant but it is safe)